### ISSUE TITLE
consider putting index.html in root folder

### ISSUE THREAD

**Issue by caseyyee** [aframevr-userland/aframe-cli#14]
Kind of throws me in a loop when looking for files.   Maybe just in the root folder as per conventional web dev.

**Comment by cvan** [comment]
yeah, I can do that - I like that idea. it threw me off too initially.

**Comment by cvan** [comment]
I tried doing this; without making some third-party patches, it's not possible yet. But it should be fixed in `brunch` in the next version, which is landing soon.
